### PR TITLE
Update the Vietnamese Translation R.2

### PR DESCRIPTION
**Pull request by Nad-SK**
- Editor translate
- Update translation files

### CODE REVIEW

**Comment by Nad-SK** [comment]
> Hi, @Nad-SK. Could you please restrict the amount of translated lines to 400 ?
Hi, @zenseii. Now, I'm in edit mode, deleting and adding lines, editing the words in the line also causes the line to be deleted and added. Deleting an entire line still saves the number of deleted lines.
Now, to reach the limit of 400 lines, I can only delete all the lines that need to be retranslated, and then translate them in the next PR.

**Comment by Nad-SK** [comment]
Thanks @zenseii for your help!

**Comment by ihhub** [comment]
Hi @Nad-SK , please address given comments by @zenseii .

File: files/lang/vi.po [hunk 1]
```diff
@@ -2693,15 +2687,13 @@ msgstr "không có bảo vật."
...
-#, fuzzy
 msgid ""
 "'Spread' combat formation spreads the castle's units from the top to the "
 "bottom of the battlefield, with at least one empty space between each unit."
 msgstr ""
 "Đội hình chiến đấu 'Dàn trải' dàn quân của ngươi từ trên xuống dưới của "
 "chiến trường, với ít nhất một khoảng trống giữa mỗi quân."
 
-#, fuzzy
 msgid ""
```
[the remark this answers is not in the document]
Comment (Nad-SK) [reply files/lang/vi.po:2731]: Sorry for the late reply.
It's fixed.

**Comment by Nad-SK** [comment]
> Hi @Nad-SK , please address given comments by @zenseii .

It's fixed.
I've been so busy these past few months.

**Comment by ihhub** [comment]
Hi @Nad-SK , your changes are breaking the compilation. Please take a look at logs which indicate the problem.

**Comment by Nad-SK** [comment]
> Hi, @Nad-SK. Could you please solve the merge conflicts?
Is this OK?

**Comment by Nad-SK** [comment]
@zenseii , Pls check

**Comment by Nad-SK** [comment]
Close for the new one.

**Pull request closed without merging**